volestipy/volestipy/include/bindings.h

```diff
@@ -56,21 +56,21 @@ class HPolytopeCPP{
       double compute_volume(char* vol_method, char* walk_method, int walk_len, double epsilon, int seed);
 
       // the generate_samples() function
-      double generate_samples(int walk_len, int number_of_points, int number_of_points_to_burn, bool boundary, 
-       bool cdhr, bool rdhr, bool gaussian, bool set_L, bool accelerated_billiard, bool billiard, bool ball_walk, double a, double L,  
+      double generate_samples(int walk_len, int number_of_points, int number_of_points_to_burn, bool boundary,
```

Comment:
there are a few changes here and there, probably by your editor. It's ok in general but I think they pollute the history without reason. It would be cleaner to remove them.